src/syscall_handler.c

```diff
@@ -125,6 +128,25 @@ int sys_chdir(const char *ns, const char *name)
 	return process_chdir(current, ns, name);
 }
 
+int sys_mkdir(const char *name){
+	struct fs_dirent *cwd = current->cwd;
+	return fs_dirent_mkdir(cwd, name);
+}
+
+int sys_readdir(const char *name, char *buffer, int len){
+	struct fs_dirent *cwd = current->cwd;
+	struct fs_dirent *d = fs_dirent_namei(cwd, name);
+	if (!d) return -1;
```

Comment:
Do we need to pass back a specific errno?  (I think John is working on a branch where errno is now defined.)